src/errors/errors.go

```diff
@@ -14,6 +14,8 @@ var (
 	// KTS_USER_NOT_FOUND is used to indicate that the requested user was not found
 	KTS_USER_NOT_FOUND = &models.KTSError{ErrorMessage: "USER_NOT_FOUND", Status: 404}
 	// KTS_NOT_FOUND is used to indicate that the requested resource was not found
+	KTS_MOVIE_NOT_FOUND = &models.KTSError{ErrorMessage: "MOVIE_NOT_FOUND", Status: 404}
```

Comment:
Why not just use NOT_FOUND?

Reply:
For a more specific detail on what went wrong

Reply:
updated to NOT_FOUND